packages/react-accessible-tooltip/src/Tooltip.js

```diff
@@ -11,9 +11,6 @@ export type LabelProps = {
         onFocus: () => void,
     },
     isHidden: boolean,
-    requestHide: () => void,
```

Comment:
Dropping support. Tooltips shouldn't really have any interactive content, so these should be redundant anyway.